client/docs/useful-commands.md

```diff
@@ -0,0 +1,19 @@
+```
+$ npm run update         Install dependencies, clean packages, boostrap, then build.
+$ npm i                  Install the root's packages. Be sure to run 'npm run bootstrap'.
```

Comment:
```suggestion
$ npm i                  Install the root's packages. Be sure to run 'npm run bootstrap' first.
```
?

Reply:
No, you run it afterwards. Bootstrap requires that Lerna itself be installed already, and Lerna itself needs access to the installed modules.

Reply:
Okay, in that case:
```suggestion
$ npm i                  Install the root's packages. Be sure to run 'npm run bootstrap' afterwards.
```